tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -1917,5 +1917,70 @@ public void RaisesAppDomainUnhandledExceptionEvent (ApplePlatform platform)
 				var output = ExecuteWithMagicWordAndAssert (appExecutable, env);
 			}
 		}
+
+		bool FindAssembly (string path, string dllName)
+		{
+			foreach (string file in Directory.GetFiles (path, "*.dll", SearchOption.AllDirectories)) {
+				if (Path.GetFileName (file).Equals (dllName, StringComparison.OrdinalIgnoreCase)) {
+					return true;
+				}
+			}
+
+			return false;
+		}
+
+		[Test]
+		[TestCase (ApplePlatform.iOS, "ios-arm64;")]
+		[TestCase (ApplePlatform.MacOSX, "osx-arm64;osx-x64")]
+		[TestCase (ApplePlatform.MacCatalyst, "maccatalyst-x64;")]
```

Comment:
Shouldn't we test `maccatalyst-arm64` as well?

Reply:
Added. I didn't add simulators since we already test this on physical devices.